app/services/catalog/collect_local_tag_resources.rb

```diff
@@ -0,0 +1,37 @@
+module Catalog
```

Comment:
I wonder if this should move to a `Tag` namespace? Organizing our future tag based services would make a lot of sense in their own namespace.